plugins/utils/obsidian/options.nix

```diff
@@ -203,11 +203,14 @@ with lib;
   '';
 
   completion = {
-    nvim_cmp = helpers.mkNullOrOption types.bool ''
-      Set to false to disable completion.
-
-      Default: `true` if `nvim-cmp` is enabled (`plugins.cmp.enable`).
-    '';
+    # FIXME should this accept raw types?
```

Comment:
Isn't this already the case ?

Reply:
No, `mkNullOrOption` does not apply `maybeRaw` like the `defaultNullOpts` helpers do.